grammars/perl 6.cson

```diff
@@ -0,0 +1,305 @@
+fileTypes: [
+  "p6"
+  "pl"
```

Comment:
This will clash with the non-perl 6 file types in other grammar since it also registers for `pl` as well I believe.

Perhaps we can just use the file types that are explicitly for perl 6 initially?